tests/sles4sap/hana_cluster.pm

```diff
@@ -52,18 +68,21 @@ sub run {
         foreach ($node1, $node2) {
             add_to_known_hosts($_);
         }
-        assert_script_run "scp -qr /usr/sap/$sid/SYS/global/security/rsecssfs/* root\@$node2:/usr/sap/$sid/SYS/global/security/rsecssfs/";
+        assert_script_run "scp -qr /usr/sap/${sid}/SYS/global/security/rsecssfs/* root\@${node2}:/usr/sap/${sid}/SYS/global/security/rsecssfs/";
         assert_script_run qq(su - $sapadm -c "hdbsql -u system -p $sles4sap::instance_password -i $instance_id -d SYSTEMDB \\"BACKUP DATA FOR FULL SYSTEM USING FILE ('backup')\\""), 900;
         assert_script_run "su - $sapadm -c 'hdbnsutil -sr_enable --name=$node1'";
 
         # Synchronize the nodes
         barrier_wait "HANA_INIT_CONF_$cluster_name";
         barrier_wait "HANA_CREATED_CONF_$cluster_name";
 
-        # Upload the configuration into the cluster
+        hanasr_angi_hadr_providers_setup($sid, $instance_id, $sapadm) if get_var('USE_SAP_HANA_SR_ANGI');
+
+        # Commits configuration changes into the cluster
+        my $resource = get_var('USE_SAP_HANA_SR_ANGI') ? "mst_SAPHanaCtl_${sid}_HDB$instance_id" : "msl_SAPHana_${sid}_HDB$instance_id";
```

Comment:
same code of https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/19867/files#diff-0ce6bf27b6d63ed8ecf5424eceba9f70b43eda0240b6078fd2b78e6dd66f9e7cR954

Reply:
You mean that I could use `$hana_resource` filled in lib here?

Reply:
maybe yes, a lib function to return the list or resources. This function internally deal with the USE_SAP_HANA_SR_ANGI to decide what to return.